### DISCUSSION TITLE
.webp support

### DISCUSSION THREAD

**Discussion by n8te-the-gr8te** [GrandyB/SheetsIO#62] [Ideas]
Hi Grandy. Thanks so much for making this useful tool. The images I use are in .webp by default. Any plans to add support for .webp images in the future? Would save me time having to convert images to other formats.

**Comment by GrandyB** [comment]
Sounds like a perfectly reasonable feature request to me; thanks for sending it across! 

I am in the middle of rewriting large parts of the backend of the application but after that's done I'll look into things like this. Looking at weeks/months kind of range in theory.

**Reply by n8te-the-gr8te** [reply]
ok so funny story but i just tried it and even with the json expecting png, webp works :O

**Reply by GrandyB** [reply]
Marvellous ~~bug~~ feature.
Out of interest does it store locally in webp or in PNG?

**Reply by n8te-the-gr8te** [reply]
png

**Reply by Awmusic12635** [reply]
Not sure if there is something different with my setup but webp images did not work for me when using the image type. It complained about it being an unsupported type and listed out the expected possible formats.

**Reply by GrandyB** [reply]
Would you be able to attach/link an example just so I have the same thing to work off of? that'd be grand!

**Reply by Awmusic12635** [reply]
https://lmsassets.consolegamingleague.com/media/teamlogos/IMG_6716.webp is the image

Below is the error
![image](https://github.com/user-attachments/assets/16f86ee7-93da-4bfa-8808-b44db85f62bf)

The config in the config file is

```{ "cell": "D2", "name": "Team1Logo", "fileExtension": "png" },```

**Reply by GrandyB** [reply]
Thanks! 

I'll report back in a couple weeks once I'm home and can attempt recreation.

**Comment by n8te-the-gr8te** [comment]
Not sure how much this matters but the images i’m using are hosted online, not local files. Here’s one examplehttps://sendou.nyc3.cdn.digitaloceanspaces.com/NezG8q-KDo7ZZu1bMV4xn-1712552610741.webpOn May 6, 2025, at 3:20 AM, Mark Bishop ***@***.***> wrote:﻿
Thanks!
I'll report back in a couple weeks once I'm home and can attempt recreation.

**Comment by n8te-the-gr8te** [comment]
Woops email did weird stuff. This is the link
 
https://sendou.nyc3.cdn.digitaloceanspaces.com/NezG8q-KDo7ZZu1bMV4xn-1712552610741.webp
